spec/signable/query/client_spec.rb

```diff
@@ -0,0 +1,259 @@
+require "spec_helper"
+
+describe Signable::Query::Client do
+  include_context 'when signable is configured'
+
+  EXISTENT_FINGERPRINT = '83ed33d1844c54a053233d00e6ab6e33'.freeze
+
+  def client
+    described_class.new
+  end
+
+  def create_envelope_info(template_fingerprint:, party_id:)
+    template = client.find('templates', template_fingerprint)
+    create_info = double(form_data: {
+      envelope_title: 'whatever',
+      envelope_documents: [
+        {
+          document_title: 'title',
+          document_template_fingerprint: template_fingerprint,
+        }
+      ],
+      envelope_parties: [
+        {
+          party_name: 'something',
+          party_email: 'something@gmail.com',
+          party_id: party_id,
+        }
+      ]
+    })
+  end
+
+  describe '#all' do
+    context 'when requesting all envelopes' do
+      it 'gets a response listing all envelopes', vcr: 'client/envelopes/all' do
```

Comment:
```suggestion
      it 'gets a response listing all envelopes', :aggregate_failures, vcr: 'client/envelopes/all' do
```
Same suggestion for the other it blocks with multiple expects 🙂

Reply:
I've added an `:aggregate_failures` at the top of the file.